### PR TITLE
KAFKA-8857; Don't check synonyms while determining if config is readOnly

### PR DESCRIPTION
**Pull request by rajinisivaram**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala [hunk 1]
```diff
@@ -159,9 +161,10 @@ class DynamicBrokerReconfigurationTest extends ZooKeeperTestHarness with SaslSet
   }
 
   @Test
-  def testKeyStoreDescribeUsingAdminClient(): Unit = {
+  def testConfigDescribeUsingAdminClient(): Unit = {
 
-    def verifyConfig(configName: String, configEntry: ConfigEntry, isSensitive: Boolean, expectedProps: Properties): Unit = {
+    def verifyConfig(configName: String, configEntry: ConfigEntry, isSensitive: Boolean, isReadOnly: Boolean,
```
Comment (omkreddy) [inline core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala:166]: isReadOnly flag is unused?

Comment (rajinisivaram) [reply core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala:166]: Oops! Good catch @omkreddy, thanks. Fixed.

**Review by omkreddy — APPROVED** [review]
@rajinisivaram Thanks for the PR. LGTM.

**Comment by rajinisivaram** [comment]
@omkreddy Thanks for the review, merging to trunk.

**Pull request merged**